Dockerfile

```diff
@@ -9,7 +9,11 @@ ENV DEBIAN_FRONTEND noninteractive \
     SERVER_URL https://localhost:4443 \
     RUNDECK_STORAGE_PROVIDER file \
     RUNDECK_PROJECT_STORAGE_TYPE file \
-    NO_LOCAL_MYSQL false
+    NO_LOCAL_MYSQL false \
+    LOGIN_MODULE RDpropertyfilelogin \
+    JASS_CONF_FILE jass-loginmodule.conf \
+    KEYSTORE_PASS adminadmin \
```

Comment:
This is to use 3rd party ssl cert. Will require truststore and keystore to be volume mounted at the correct location